cmd/skupper/skupper_kube_link.go

```diff
@@ -149,6 +176,9 @@ func (s *SkupperKubeLink) Status(cmd *cobra.Command, args []string) error {
 				fmt.Println(err)
 				break
 			} else if allConnected(links) || i == waitFor {
+				fmt.Println("\nLinks initiated from this site:")
+				fmt.Println("-------------------------------")
+
 				if len(links) == 0 {
 					fmt.Println("There are no links configured or active")
```

Comment:
Is 'configured OR active' correct?
what does 'configured' mean, 'created from this site'?

Reply:
A link could be not configured if there was an error when creating the link, like "Failed to redeem claim".

A configured link can be active or not active. An active link transitions to  a "not active" status when the site that points to has revoked all access, for example.